### PR TITLE
Hello world page #1

### PR DESCRIPTION
**Pull request by otabaran**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<HTML>
```
Comment (cristigar) [inline index.html]: HTML specification requires to indicate the `DOCTYPE` declaration https://html.spec.whatwg.org/#the-doctype.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+<HEAD>
+
+<TITLE></TITLE>
+
+</HEAD>
+
+<BODY>
```
Comment (cristigar) [inline index.html]: It's preferable to use lower-case for tag names, as in this example from the spec https://html.spec.whatwg.org/#a-quick-introduction-to-html.

File: index.html (hunk 1 above)
Comment (DimaSocolovschi) [inline index.html]: <!DOCTYPE HTML>

Comment (DimaSocolovschi) [inline index.html]: Before "HTML" tag add "!DOCTYPE HTML"

File: index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<HTML>
+
+<HEAD>
+
+<TITLE></TITLE>
```
Comment (DimaSocolovschi) [inline index.html]: Add title for our site.

**Review by DimaSocolovschi — COMMENTED** [review]
Check the code

**Review by DimaSocolovschi — APPROVED** [review]

**Review by cristigar — CHANGES_REQUESTED** [review]

**Pull request merged**